apps/wallet/src/app/core/services/ledger.service.ts

```diff
@@ -0,0 +1,286 @@
+import { Platform } from '@angular/cdk/platform';
+import { Injectable, ClassProvider, Inject } from '@angular/core';
+
+import { BehaviorSubject } from 'rxjs';
+import TransportU2F from '@ledgerhq/hw-transport-u2f';
+import TransportWebUSB from '@ledgerhq/hw-transport-webusb';
+import { WavesLedger } from 'lto-ledger-js-unofficial-test';
+
+import { TransactionTypes } from '@wallet/core/transaction-types';
+import { LTO_NETWORK_BYTE, LTO_PUBLIC_API } from '@wallet/tokens';
+
+import {
```

Comment:
sponsor as well?

Reply:
AFAIK the wallet doesn't support sponsor transactions (yet at least), neither does the Ledger app